config/config_ussdapp.qa.json

```diff
@@ -25,6 +25,10 @@
     "location_api_url": "http://rr.praekeltfoundation.org/locationfinder/",
     "poi_types": [],
     "template": "Nearby services: {{ results }}.",
+    "welcome_message_sets": {
+        "refugee": "2",
+        "migrant": "3"
```

Comment:
These are strings, not numbers. Give the test failures you were having issues with, perhaps they should be numbers? So `2` and `3` instead of `"2"` and `"3"`.